src/SDK.ts

```diff
@@ -881,17 +705,16 @@ export default class SDK implements ISDK {
         { SurveyOwnerId: surveyOwnerId },
         "Get Survey Invite Link Started");
     }
```

Comment:
Does this need to be refactored?

Reply:
Discussed that refractoring is not needed